Sources/DistributedCluster/Cluster/Downing/DowningStrategy.swift

```diff
@@ -93,10 +93,10 @@ internal distributed actor DowningStrategyShell {
         self.strategy = strategy
         self.actorSystem = system
         self.eventsListeningTask = Task { [weak self] in
-            guard let __secretlyKnownToBeLocal = self else { return } // FIXME(local): we really need `local` here
-
-            for await event in system.cluster.events {
-                try __secretlyKnownToBeLocal.receiveClusterEvent(event)
+            try await self?.whenLocal { __secretlyKnownToBeLocal in
```

Comment:
No need for the `__secretlyKnownToBeLocal` because `whenLocal` does bring the right isolation.

`__secretlyKnownToBeLocal` is a hack that turns off checking and we have to remove it.